contentcuration/contentcuration/frontend/shared/views/GlobalSnackbar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+  <v-snackbar
+    v-if="snackbarIsVisible"
+    :key="key"
+    :timeout="snackbarOptions.duration"
+    bottom
+    :value="snackbarIsVisible"
+  >
+    {{ snackbarOptions.text }}
+    <v-btn
```

Comment:
Sorry, just one small change- could you add `v-if="actionText"` to handle if none is provided? Some snackbars might be really short messages like "Copied successfully!" that should be auto-dismissed

Reply:
Done and added